docs/Configurable.md

```diff
@@ -58,7 +58,7 @@ const TextBlockEditor = () => {
     );
 };
 ```
- 
+This hook is only used within a Configurable component.
```

Comment:
This line is redundant with the information given above. IMO it's not necessary to add it.